magento/magento2
# Design issue fixed in category page magento2#30397

### Description (*)
**Fixed #30397 count design issue fixed**

Preconditions (*)
PHP 7.4.11
magento2-2.4

Manual testing scenarios (*)
1.Open magento base url.
2.Go to category page.
3.Go to Shopping Options on left side of page.
4.Open any CATEGORY/PRICE dropdown.
5.Check count design.

Expected result (*)
https://user-images.githubusercontent.com/50991251/95648439-c8acd500-0af4-11eb-9517-0108d76889f6.png

Actual result (*)
https://user-images.githubusercontent.com/50991251/95648477-1d505000-0af5-11eb-956c-154f3bf1c189.png
